bundles/org.openhab.binding.innogysmarthome/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -25,6 +26,12 @@
 		<label>Blinds Position</label>
 		<description>Controls the blinds</description>
 		<category>Blinds</category>
+		<config-description>
+			<parameter name="invert" type="boolean">
```

Comment:
I am thinking about if "invert-rollershutter" is a valid candidate for a default profile in OHC.

Reply:
These could be applied to other types as well I guess. But if I'm correct there can only be 1 profile per item? In which case using a profile might not be the best approach. Also profiles are bound to items not channels (I'm not really sure if that is true, I guess I should read more about profiles :slightly_smiling_face: ) But if it's bound to items the profile approach might not be optimal. Maybe some kind of default channel parameter on certain channels, if technically possible?

Reply:
The number of profiles is limited to one. A profile is correlated to Links between Items and Channels. They can either manipulate the state of the Item as well as the command towards the Channel (see [Developing a Profile](https://www.openhab.org/docs/developer/transformations/#developing-a-profile) for more information).

In general an "invert" profile could be possible for other types too. But I do not think it makes sense for all of them.

Another option would be to introduce a system default channel type for rollershutter / blinds. This could e.g. use a default configuration like yours. But we have to make sure that every time the channel type will be used one has to implement the business logic for the inversion too. That will be quite error-prone.

Reply:
I submitted a draft for my suggested a profile https://github.com/openhab/openhab-core/pull/1245.